### ISSUE TITLE
Autoformat not working on save

### ISSUE THREAD

**Issue by jrock2004** [prettier/vim-prettier#191]
**Do you want to request a *feature* or report a *bug*?**
bug

**What is the current/expected behavior?**
Just installed the plugin and when I save the file, nothing changes. If I do `:Prettier` it then formats my document. I ran `echo g:prettier#autoformat` and it returns a `1`

**What version of `vim-prettier` are you using - (output of `:PrettierVersion`) ?**
0.2.7

**What version of `prettier` are you using - (output of  `:PrettierCliVersion`) ?**
1.18.2

**What is your `prettier` executable path - (output of  `:PrettierCliPath`) ?**
`/Users/jcostanzo/.dotfiles/config/nvim/plugged/vim-prettier/node_modules/.bin/prettier`

**Did this work in previous versions of vim-prettier and/or prettier ?**
This is my first time using this so its unknown

**Comment by lfarroco** [comment]
look for the following text in the README:
> o enable vim-prettier to run in files without requiring ...
there are some options to enable autosaving (the code snippets should be added to the `~/.vimrc` file)

**Comment by vadimshvetsov** [comment]
The docs are not clear for me. It took 15 mins to make it works on autosave. It's not easy to understand, that options below must be used **together** for execute prettier on autosave without "@format" or "@prettier" tags.
```vim
let g:prettier#autoformat = 1
let g:prettier#autoformat_require_pragma = 0
```
Looks reasonable to use only second command without first.

**Comment by drjosephliu** [comment]
I agree with @vadimshvetsov . This should be more clear on the readme

**Comment by mitermayer** [comment]
@drjosephliu Please feel free to submit a PR for it we would love contributions!
